x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/multi_page_layout/components/confirm_incoming_data_with_preview.tsx

```diff
@@ -54,7 +54,7 @@ const DATA_PREVIEW_OMIT_KEYS = [
 ];
 
 const CleanOverflowDescriptionList = styled(EuiDescriptionList)`
-  overflow: hidden;
+  overflow: scroll;
```

Comment:
Allow users scroll through the content for large documents

Reply:
Hi! Originally in the designs it was intentional that this was just a limited snapshot of the documents, would you be able to record a video of what the scroll looks like for larger documents and we can get @dborodyansky to check he's happy with it? Thanks 👍

Reply:
@hop-dev  Here it is! Let me know if that make sense or not

![scroll_endpoint_events](https://user-images.githubusercontent.com/15727784/191928151-44b5d758-f349-47ed-ba64-4301aabad63e.gif)

cc: @dborodyansky